doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst

```diff
@@ -135,60 +269,43 @@ There are two downsides to the implementation above for C functions:
 2.  To follow current implementation standards, documentation should be attached
     directly to the Python source code.
 
-Alternatively for C functions, the implementation of ``like=`` could be moved
-into the C implementation itself. This is not the primary suggestion here due
-to its inherent complexity which would be difficult too long to describe in its
-entirety here, and too tedious for the reader. However, we leave that as an
-option open for discussion.
+The first version of this proposal suggested the C implementation above as one
+viable solution. However, due to the downsides pointed above we have decided to
+implement that entirely in C. Please refer to [implementation]_ for details.
 
-Usage
------
+Alternatives
+------------
 
-The purpose of this NEP is to keep things simple. Similarly, we can exemplify
-the usage of ``like=`` in a simple way. Imagine you have an array of ones
-created by a downstream library, such as CuPy. What you need now is a new array
-that can be created using the NumPy API, but that will in fact be created by
-the downstream library, a simple way to achieve that is shown below.
+Recently a new protocol to replace ``__array_function__`` entirely was proposed
```

Comment:
Whoa I didn't know this. Do we need any comparison over NEP 37 at this point? I know I am overreaching but sounds really interesting. I'll read that one after this.

Reply:
Yup, the elephant in the room was never the syntax here :). NEP 37 is good context (I personally think the appendix may help understand some of the considerations). There are also these meeting notes (a lot in there was not actually discussed), but I am not sure it is approachable: https://github.com/numpy/archive/blob/master/other_meetings/2020-04-21-array-protocols_discussion_and_notes.md

I made https://github.com/seberg/numpy-dispatch with the hope we can get try the NEP 37 idea in libraries, I guess @shoyer might be working on a JAX prototype?

Well, this is just context for you though, if you are curious to dive into it.  In the end, I personally hope we can rope in library authors to help figure out what the right approach is probably, and the having something to play with helps, both with getting a feel of things and getting enthusiasm for experimenting.  (But that is just my sloppy way of saying it/take on this.)